src/contract/polls.cpp

```diff
@@ -465,6 +466,118 @@ double GetMoneySupplyFactor()
     return Factor;
 }
 
+Array GetJSONPolls(bool bDetail, bool includeExpired, std::string byTitle)
+{
+    Array aPolls;
+    Object oPoll;
```

Comment:
Move the temporaries into the for loops and we can drop the `clear` and `shrink` calls.